api/build.go

```diff
@@ -362,6 +362,72 @@ func GetBuilds(c *gin.Context) {
 	c.JSON(http.StatusOK, b)
 }
 
+// GetBuildOrgs represents the API handler to capture a
+// list of builds associated with an org from the configured backend.
+func GetBuildOrgs(c *gin.Context) {
```

Comment:
Can you add the open api spec for this API endpoint? we should also rebuild the spec to contain these changes.

[There are plenty of examples in this file](https://github.com/go-vela/server/blob/78b8d02d03ad136dbea2a8e87ea4d9768e436338/api/build.go#L260-L297) let me know if you have questions or aren't sure how to build it.

Reply:
Just tacking on... should we name this function something a bit more intuitive?
like.... GetOrgBuilds or GetBuildsOrg?

To me, GetBuildOrgs implies you're getting a list of orgs based on build, or you're getting a single build based on multiple orgs? Maybe im just being picky

Reply:
How about GetBuildsByOrg?

Reply:
My vote would be GetOrgBuilds since `GetBuildsByOrg` could be confused with GetBuilds grouped by Org.

Reply:
Changed to GetOrgBuilds.